### PR TITLE
View an individual article

### PR DESCRIPTION
**Pull request by holyseo**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 
 function App() {
   return (
     <div className="container">
       <Header />
-      <AllArticles />
+      <Routes>
+        <Route path="/" element={<AllArticles />} />
+        <Route
+          path="/articles/:article_id/comments"
```
Comment (popatre) [inline src/App.jsx]: This url doesnt make sense

**Review by popatre — CHANGES_REQUESTED** [review]
- Good linking to the article from the articles list - however the url doesnt really make sense
- [ ] Need to add some  isLoading pattern to give some user feedback that something happening, especially on a slow connection or slow server connection
- [ ] The url is /articles/id/comments - however, we're just viewing a single card, so the comments bit doesnt make sense. 
- [ ] We're missing the main details of the article. It should have a body with the details on the article. Also, you have a title of article id, but the articles have a title which would make more sense to use instead of the article id

File: src/components/ArticleCard.jsx [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
 const ArticleCard = ({ article }) => {
   return (
     <div className="article_cards">
-      <h4>{article.title}</h4>
+      <Link to={`/articles/${article.article_id}`}>
+        <h4>{article.title}</h4>
+      </Link>
       <p>Topic = {article.topic}</p>
       <p>Author = {article.author}</p>
       <p>Created at = {article.created_at}</p>
```
Comment (nc-modou) [inline src/components/ArticleCard.jsx:11]: Take a look at the toUTCstring method to format this date in a more readable way

**Review by nc-modou — COMMENTED** [review]
Responsed well to jonathan's feedback about the url and loading state, but you're still showing the article ID which means nothing to the user - that should be hidden and just used behind the scenes.

After fixing that, and addressing comments, feel free to merge

**Pull request merged**